src/core/pages/PatientSummary/index.js

```diff
@@ -38,43 +35,48 @@ const styles = theme => ({
         flexDirection: "column",
         height: 100,
         backgroundColor: theme.palette.mainColor,
-        background: theme.patientSummaryPanel.topBlock.background,
-        backgroundSize: "cover",
         justifyContent: "center",
         alignItems: "center",
+        position: "relative",
         color: "#fff",
         '&:hover': {
             cursor: "pointer",
-        }
+        },
     },
     icon: {
         marginBottom: 10,
+        zIndex: 99999999,
```

Comment:
bad practice. Find the smallest z-index that achieves the effect you want, because you'll always need to put something in front of that icon (e.g. a dialog)